BlocNotes/BlocNotes/DetailNoteViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  DetailNoteViewController.m
+//  BlocNotes
+//
+//  Created by Eddy Chan on 6/26/16.
+//  Copyright © 2016 chan-e. All rights reserved.
+//
+
+#import "DetailNoteViewController.h"
+#import "CoreDataStack.h"
+#import "Note.h"
+
+@interface DetailNoteViewController () <UITextViewDelegate>
+
+@property (weak, nonatomic) IBOutlet UILabel *placeholderTextLabel;
+@property (weak, nonatomic) IBOutlet UITextView *bodyTextView;
+
+@end
+
+@implementation DetailNoteViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    self.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc]
```

Comment:
What is the purpose of this? Do you just want to hide the back button? 
You could either present the view modally versus pushing it or you just set it to `nil`